tests/test_observables.py

```diff
@@ -0,0 +1,81 @@
+"""Tests related to petab.observables"""
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Similar lines in 2 files](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4973405)